thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -579,20 +582,34 @@ public Response evaluateAlertFilterByAutoTuneId(@PathParam("autotuneId") long id
   @POST
   @Path("replay/function/{id}")
   public Response anomalyFunctionReplay(@PathParam("id") @NotNull long functionId,
-      @QueryParam("start") @NotNull String replayStartTimeIso, @QueryParam("end") @NotNull String replayEndTimeIso,
+      @QueryParam("time") String replayTimeIso, @QueryParam("duration") @DefaultValue("30") int replayDuration,
```

Comment:
Does this mean that you will fix the end to now? This is less flexible than the original one. Why making this choice?